ios/browser/api/brave_shields/brave_shields_utils_ios+private.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_API_BRAVE_SHIELDS_BRAVE_SHIELDS_UTILS_IOS_PRIVATE_H_
+#define BRAVE_IOS_BROWSER_API_BRAVE_SHIELDS_BRAVE_SHIELDS_UTILS_IOS_PRIVATE_H_
+
+#import <Foundation/Foundation.h>
+
+#include "brave/ios/browser/api/brave_shields/brave_shields_utils_ios.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+class ProfileIOS;
+
+@interface BraveShieldsUtilsIOS (Private)
```

Comment:
moving forward can we put this kind of stuff in `brave/components/{component_name}/ios` (alongside core/content)?

Reply:
if that causes some problem we can leave it for now, but I want to look at this stuff in general and try to keep as much as possible under components

Reply:
I've updated the iOS code to `brave/components/brave_shields/ios` 👍. Small factory to return it in `ios/browser/brave_shields` as that needs `ios/chrome`.